External/libgit2

```diff
@@ -1 +1 @@
-Subproject commit c27b4afcdd80f5a45d7120044bf3d78272181abb
+Subproject commit abbe17ef1ffbc264101808829a3317c1c805df96
```

Comment:
This is still reverting libgit2 to an older version. You should be able to follow the instructions [here](https://github.com/libgit2/objective-git/pull/513#issuecomment-134785288) to push a new commit to this PR.